src/main/java/com/rapid7/ias/bamboo/impl/InsightAppSecScanTask.java

```diff
@@ -54,6 +58,11 @@ public TaskResult execute(CommonTaskContext taskContext) throws TaskException {
         appName = configMap.get(APP_NAME);
         scanConfigName = configMap.get(SCAN_CONFIG_NAME);
 
+        proxyHost = configMap.get(PROXY_HOST);
+        proxyPort = configMap.get(PROXY_PORT);
+
+        debugging = configMap.get(debugging);
```

Comment:
Why don't you have a constant for the debugging property name, like with PROXY_HOST and PROXY_PORT?